Explorer/Assets/DCL/Character/CharacterPreview/CharacterPreviewController.cs

```diff
@@ -31,15 +33,19 @@ namespace DCL.CharacterPreview
 
         public CharacterPreviewController(World world, CharacterPreviewAvatarContainer avatarContainer,
             CharacterPreviewInputEventBus inputEventBus, IComponentPool<CharacterPreviewAvatarContainer> characterPreviewContainerPool,
-            CharacterPreviewCameraSettings cameraSettings)
+            CharacterPreviewCameraSettings cameraSettings, IComponentPool<Transform> transformPool)
         {
             globalWorld = world;
             characterPreviewAvatarContainer = avatarContainer;
             cameraController = new CharacterPreviewCameraController(inputEventBus, characterPreviewAvatarContainer, cameraSettings);
             this.characterPreviewContainerPool = characterPreviewContainerPool;
 
+            var parent = transformPool.Get();
+            parent.SetParent(avatarContainer.avatarParent, false);
```

Comment:
Does the transform need resetting?

Reply:
I guess that we don't move the avatars in the local position but I think an additional check wouldn't hurt

Reply:
I did reset here just in case, but also in the pool itself `onRelease`